From 1c0ae75a84ea4efeff4f92abcf16728a516f6b28 Mon Sep 17 00:00:00 2001
From: Hui Xia <hui.xia@intel.com>
Date: Tue, 3 Jul 2018 00:29:05 -0400
Subject: [PATCH 1691/2302] media: platform: intel: complete description I2C_ID

The configure INTEL_IPU4_ADV7481_I2C_ID can not accept
setting value due to missing range parameter. Add range
parameter for it.

Signed-off-by: Hui Xia <hui.xia@intel.com>
---
 drivers/media/platform/intel/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/drivers/media/platform/intel/Kconfig b/drivers/media/platform/intel/Kconfig
index d198d4e..2cd9d2f 100755
--- a/drivers/media/platform/intel/Kconfig
+++ b/drivers/media/platform/intel/Kconfig
@@ -100,7 +100,8 @@ config INTEL_IPU4_ADV7481
 	HDMI2MIPI convertor device ADV7481
 
 config INTEL_IPU4_ADV7481_I2C_ID
-	int
+	int "I2C bus ID for ADV7481"
+	range 0 8
 	default 0
 	---help---
 	I2C bus number of ADV7481 Mondello
-- 
2.7.4

